src/connector/jdbc/src/main/java/com/taosdata/jdbc/ws/JSONResultSet.java

```diff
@@ -0,0 +1,593 @@
+                    long epochSec = value / 1000_000L;
+                    long nanoAdjustment = value % 1000_000L * 1000L;
+                    return Timestamp.from(Instant.ofEpochSecond(epochSec, nanoAdjustment));
+                }
+                if (value >= 1_000_000_000_000_000_0l) {
+                    this.timestampPrecision = TimestampPrecision.NS;
+                    long epochSec = value / 1000_000_000L;
+                    long nanoAdjustment = value % 1000_000_000L;
+                    return Timestamp.from(Instant.ofEpochSecond(epochSec, nanoAdjustment));
+                }
+            }
+            case UTC: {
+                String value = row.getString(colIndex);
+                if (value.lastIndexOf(":") > 19) {
+                    ZonedDateTime parse = ZonedDateTime.parse(value, rfc3339Parser);
+                    long nanoAdjustment;
+                    if (value.length() > 32) {
+                        // ns timestamp: yyyy-MM-ddTHH:mm:ss.SSSSSSSSS+0x:00
+                        this.timestampPrecision = TimestampPrecision.NS;
+                    } else if (value.length() > 29) {
+                        // ms timestamp: yyyy-MM-ddTHH:mm:ss.SSSSSS+0x:00
+                        this.timestampPrecision = TimestampPrecision.US;
+                    } else {
+                        // ms timestamp: yyyy-MM-ddTHH:mm:ss.SSS+0x:00
+                        this.timestampPrecision = TimestampPrecision.MS;
+                    }
+                    return Timestamp.from(parse.toInstant());
+                } else {
+                    long epochSec = Timestamp.valueOf(value.substring(0, 19).replace("T", " ")).getTime() / 1000;
+                    int fractionalSec = Integer.parseInt(value.substring(20, value.length() - 5));
+                    long nanoAdjustment;
+                    if (value.length() > 32) {
+                        // ns timestamp: yyyy-MM-ddTHH:mm:ss.SSSSSSSSS+0x00
+                        nanoAdjustment = fractionalSec;
+                        this.timestampPrecision = TimestampPrecision.NS;
+                    } else if (value.length() > 29) {
+                        // ms timestamp: yyyy-MM-ddTHH:mm:ss.SSSSSS+0x00
+                        nanoAdjustment = fractionalSec * 1000L;
+                        this.timestampPrecision = TimestampPrecision.US;
+                    } else {
+                        // ms timestamp: yyyy-MM-ddTHH:mm:ss.SSS+0x00
+                        nanoAdjustment = fractionalSec * 1000_000L;
+                        this.timestampPrecision = TimestampPrecision.MS;
+                    }
+                    ZoneOffset zoneOffset = ZoneOffset.of(value.substring(value.length() - 5));
+                    Instant instant = Instant.ofEpochSecond(epochSec, nanoAdjustment).atOffset(zoneOffset).toInstant();
+                    return Timestamp.from(instant);
+                }
+            }
+            case STRING:
+            default: {
```

Comment:
indent

Reply:
This is OK. This means：If it's a STRING, the processing is the same as default